kv/local_sender.go

```diff
@@ -157,7 +159,13 @@ func (ls *LocalSender) Send(ctx context.Context, call proto.Call) {
 			trace.Event("read has no clock uncertainty")
 			header.Txn.MaxTimestamp = header.Txn.Timestamp
 		}
-		err = store.ExecuteCmd(ctx, call)
+		reply, err = store.ExecuteCmd(ctx, call.Args)
```

Comment:
If you used `:=` here, `go vet` (without --shadow) should give a warning about `err declared but not used`.